cpp/src_prims/linalg/gemm.h

```diff
@@ -180,5 +180,99 @@ void gemm(const math_t *a, int n_rows_a, int n_cols_a, const math_t *b,
        beta, cublas_h, stream);
 }
 
+/**
+ * @brief A wrapper for CUBLS GEMM function designed for handling all possible 
+ * combinations of operand layouts.
+ * It computes the following equation: Z = alpha . X * Y + beta . Z
+ * @tparam T Data type of input/output matrices (float/double)
+ * @param handle cublas handle
+ * @param z output matrix of size M rows x N columns
+ * @param x input matrix of size M rows x K columns
+ * @param y input matrix of size K rows x N columns
+ * @param _M number of rows of X and Z
+ * @param _N number of rows of Y and columns of Z
+ * @param _K number of columns of X and rows of Y
+ * @param isZColMajor Storage layout of Z. true = col major, false = row major
+ * @param isXColMajor Storage layout of X. true = col major, false = row major
+ * @param isYColMajor Storage layout of Y. true = col major, false = row major
+ * @param alpha scalar
+ * @param beta scalar
+ * @{
+ */
+template <typename T>
+void gemm(cublasHandle_t handle, T *z, T *x, T *y, int _M, int _N, int _K,
+          bool isZColMajor, bool isXColMajor, int isYColMajor,
```

Comment:
```suggestion
          bool isZColMajor, bool isXColMajor, bool isYColMajor,
```

Reply:
That is a copy-paste error, good catch! Fixed now.